source/imaer-util/src/main/java/nl/overheid/aerius/util/GeometryUtil.java

```diff
@@ -127,7 +127,11 @@ private static LineString toJtsLineString(final nl.overheid.aerius.shared.domain
       final Coordinate coordinate = toJtsCoordinate(aeriusLineString.getCoordinates()[i], aeriusLineString);
       coordinates[i] = coordinate;
     }
-    return geometryFactory.createLineString(coordinates);
+    try {
+      return geometryFactory.createLineString(coordinates);
+    } catch (final IllegalArgumentException e) {
+      throw new AeriusException(ImaerExceptionReason.GEOMETRY_INVALID, e.getMessage());
```

Comment:
The use `e.getMessage()` might be a bit inconsistent: for other locations where this reason is used we use the supplied geometry. So error messages with this might end up looking a bit weird?